astrokat/observatory.py

```diff
@@ -8,19 +10,43 @@
 
 try:
     import katconf
-except ImportError:
+    # Set up configuration source
+    _config_path = '/var/kat/config'
+    _node_file = '/var/kat/node.conf'
+    _settings = {}
+    if os.path.isdir(_config_path):
+        katconf.set_config(katconf.environ(override=_config_path))
+    elif os.path.isfile(_node_file):
+        with open(_node_file, 'r') as fh:
+            _node_conf = json.loads(fh.read())
+        for _key, _val in _node_conf.items():
+            # Remove comments at the end of the line
+            _val = _val.split("#", 1)[0]
+            _settings[_key] = _val.strip()
+            if _node_conf.get("configuri", False):
+                katconf.set_config(katconf.environ(_node_conf["configuri"]))
+            else:
+                raise ValueError("Could not open node config file using configuri")
+    else:
+        raise ValueError("Could not open node config file")
+
+except (ImportError, ValueError):
     # default reference position for MKAT array
-    ref_location = 'ref, -30:42:47.4, 21:26:38.0, 1060.0, 0.0, , , 1.15'
+    _ref_location = 'ref, -30:42:47.4, 21:26:38.0, 1060.0, 0.0, , , 1.15'
```

Comment:
Please update to match the current location:  https://github.com/ska-sa/katconfig/blob/karoo/static/arrays/karoo.array_mkat.conf#L21